src/framework/filterControl/forms/MultipleSelectFilterForm.spec.jsx

```diff
@@ -137,8 +167,10 @@ describe('MultipleSelectFilterForm', () => {
       const filter = props.onAddFilter.calls.argsFor(0)[0];
 
       expect(filter.comparisonValue.length).toBe(2);
-      expect(filter.comparisonValue.indexOf('Active')).not.toBe(-1);
-      expect(filter.comparisonValue.indexOf('Archived')).not.toBe(-1);
+      expect(filter.comparisonValue.filter(e => e.name === 'Active').length
+        ).not.toBe(-1);
```

Comment:
Better:
```
expect(filter.comparisonValue.filter(
  e => e.value === 'Active').length).toBe(1);
```
Length will always be at least 0. Previously we checked that the element was present, now we need to check that there is one element satisfying the filter condition.
And it should be `e.value` instead of `e.name`.